src/ClientData/include/ClientData/TimerData.h

```diff
@@ -66,6 +66,44 @@ class TimerData final {
 
   void UpdateMaxDepth(uint32_t depth) { max_depth_ = std::max(max_depth_, depth); }
 
+  [[nodiscard]] const orbit_client_protos::TimerInfo* GetFirstAfterTime(uint64_t time,
+                                                                        uint32_t depth) const {
+    const orbit_client_data::TimerChain* chain = GetChain(depth);
+    if (chain == nullptr) return nullptr;
+
+    // TODO: do better than linear search...
```

Comment:
Can you add a bug and link in the comment?

Reply:
done.